images/bridge2/cmd/bridge/main.go

```diff
@@ -99,6 +104,70 @@ type SessionInitHandler interface {
 	HandleSessionInit(*tracks.Session)
 }
 
+type AssistantCommands struct {
+	Endpoint string
+}
+
+func (a *AssistantCommands) HandleSessionInit(sess *tracks.Session) {
```

Comment:
I started to look at the HTTP commands code from chromestage, but I think it may simplify things to leave this mainly hard-coded for this review, and then look into integrating that in a follow-up.

Reply:
ok